### PR TITLE
api: Remove unused `timetable` endpoints

### LINKED ISSUE #3510: api: `/timetable/?infra=<infra_id>` is returning a 500
### What happened?

Title endpoint is failing. This endpoint is not used by the front anymore.

### What did you expect to happen?

To get a valid response (paginated list of timetables).

### How can we reproduce it (as minimally and precisely as possible)?

build osrd in local. call the route you will get a 500.

### What operating system, browser and environment are you using?

.

### OSRD version (top right corner `Account` button > `Informations`)

.

### PR DESCRIPTION
**Pull request by bloussou**
Removing the unused and bugged endpoint from the `openapi` and code

close #3510

### CODE REVIEW

**Review by flomonster — CHANGES_REQUESTED** [review]
The List and Retrieve mixins must be also removed.

File: python/api/osrd_infra/views/timetable.py [hunk 1]
```diff
@@ -9,21 +9,13 @@
...
-
-    def get_queryset(self):
-        queryset = Timetable.objects.order_by("-pk")
-        infra = self.request.query_params.get("infra")
-        if infra is not None:
-            queryset = queryset.filter(infra__pk=infra)
-        return queryset
+    queryset = Timetable.objects.order_by("-pk")
 
     def retrieve(self, request, pk):
```
Comment (multun) [inline python/api/osrd_infra/views/timetable.py:18]: if the retrieve mixing is to be removed, the method has to go too

Comment (bloussou) [reply python/api/osrd_infra/views/timetable.py:18]: I have kept retrieve and removed create mixins :)

**Review by flomonster — APPROVED** [review]
Great!

**Review by Math-R — APPROVED** [review]

**Pull request merged**